src/feed-item.tsx

```diff
@@ -28,37 +30,41 @@ export class FeedItem extends React.Component<Properties, State> {
     const { app, metadataService, ...model } = props;
 
     this.state = {
-      ...model
+      ...model,
     };
   }
 
   static defaultProps = {
     metadataService: metadataService,
-  }
+  };
 
   componentDidMount = async () => {
     const { metadataUrl, metadataService } = this.props;
 
     if (metadataUrl) {
       const metadata = await metadataService.load(metadataUrl);
 
-      this.setState(previousState => ({ ...previousState, ...metadata }));
+      this.setState((previousState) => ({ ...previousState, ...metadata }));
     }
-  }
+  };
 
   render() {
     const { title, description, imageUrl, znsRoute } = this.state;
     const { app } = this.props;
 
     return (
-      <div className="feed-item">
-        <Link to={`/${[znsRoute, app].join("/")}`}>
-          <div className="feed-item__text-content">
-            <h3 className="feed-item__title">{title}</h3>
-            <span className="feed-item__description">{description}</span>
+      <div className='feed-item'>
+        <Link to={`/${[znsRoute, app].join('/')}`}>
+          <div className='feed-item__text-content'>
+            <h3 className='feed-item__title'>{title}</h3>
+            <span className='feed-item__description'>{description}</span>
           </div>
           <Image
-            className="feed-item__image"
+            useCloudinary
```

Comment:
to me a cloudinary image an a normal image are two totally different things. instead of using a flag that essentially just results in two totally different code paths for the component, can we just use a `CloudinaryImage` (or `CloudImage` if you want to avoid tying the name to the service) component or something here?